service/src/main/kotlin/app/cash/backfila/dashboard/GetActiveBackfillsAction.kt

```diff
@@ -0,0 +1,64 @@
+package app.cash.backfila.dashboard
+
+import app.cash.backfila.service.persistence.BackfilaDb
+import app.cash.backfila.service.persistence.BackfillRunQuery
+import app.cash.backfila.service.persistence.BackfillState
+import app.cash.backfila.service.persistence.DbBackfillRun
+import java.time.Instant
+import javax.inject.Inject
+import misk.MiskCaller
+import misk.hibernate.Query
+import misk.hibernate.Transacter
+import misk.hibernate.newQuery
+import misk.scope.ActionScoped
+import misk.security.authz.Authenticated
+import misk.web.Get
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+
+class GetActiveBackfillsAction @Inject constructor(
+  private val caller: @JvmSuppressWildcards ActionScoped<MiskCaller?>,
+  @BackfilaDb private val transacter: Transacter,
+  private val queryFactory: Query.Factory,
+) : WebAction {
+
+  @Get("/backfills/running")
```

Comment:
Probably needs to be paginated.

Reply:
yeah - was kinda assuming the number of backfills concurrently running wouldn't generally be enough to require pagination, but that's I guess that's not necessarily the case.

Reply:
Actually, looks like we don't have pagination on any of the endpoints apart from the `/services/{service}/variants/{variant}/backfill-runs` one.  (which makes sense because it's completely unbounded)

Feels like we might be able to avoid it for this endpoint given it's only returning currently running ones.  WDYT?

Reply:
https://github.com/cashapp/backfila/blob/master/service/src/main/kotlin/app/cash/backfila/dashboard/GetBackfillRunsAction.kt#L59

Reply:
You are listing runs so of course it needs to be paginated. 

The other ones might need a rethink soon given how popular Backfila has become.

Why did you pick runs to show rather than services?